internal/metastore/kv/suffix_snapshot.go

```diff
@@ -14,7 +14,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package rootcoord
+package kv
```

Comment:
We will need to think of suffix snapshot clean up later

Reply:
We will support cleaning up snapshot data from mysql metastore with high priority